src/components/AssetTypesComponent.jsx

```diff
@@ -27,6 +28,21 @@ export class AssetTypesComponent extends React.Component {
     this.props.loadAssetTypes(activePage);
   }
 
+  handleViewClick = () => {
+    this.setState({ showModal: true });
+    return <ArtModal open={this.state.showModal} />;
+  }
+
+  formatDate = (assetType, key) => {
```

Comment:
There's a utility function in https://github.com/AndelaOSP/art-dashboard/pull/68/files#diff-649b5430456ee1e3d3e44ef559647d99R1 that can be used here. Liaise with Arthur to know when he's merged to develop

Reply:
OK